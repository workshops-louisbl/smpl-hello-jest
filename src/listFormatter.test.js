import formatList from "./listFormatter";

test("format a list", () => {
  const result = formatList([
    {key: "key1", value: "value1"},
    {key: "key3", value: "value3"},
    {key: "key4", value: "value4"},
    {key: "key5", value: "value5"},
  ])

  expect(result).toMatchSnapshot();
})
