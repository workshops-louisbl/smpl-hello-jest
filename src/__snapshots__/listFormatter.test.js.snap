// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format a list 1`] = `
"* key1 => value1
* key3 => value3
* key4 => value4
* key5 => value5"
`;
